packages/protocol/releaseData/nonstandarVersionReports/baklava_fix_report_report.json

```diff
@@ -0,0 +1,859 @@
+          "patch": [
+            {
+              "contract": "Accounts",
+              "type": "DeployedBytecode"
+            },
+            {
+              "contract": "Accounts",
+              "dependency": "Signatures",
+              "type": "LibraryLinkingChange"
+            }
+          ]
+        },
+        "versionDelta": {
+          "storage": "=",
+          "major": "=",
+          "minor": "+1",
+          "patch": "0"
+        }
+      },
+      "Election": {
+        "changes": {
+          "storage": [],
+          "major": [],
+          "minor": [
+            {
+              "contract": "Election",
+              "signature": "activateForAccount(address,address)",
+              "type": "MethodAdded"
+            }
+          ],
+          "patch": [
+            {
+              "contract": "Election",
+              "type": "DeployedBytecode"
+            },
+            {
+              "contract": "Election",
+              "dependency": "AddressSortedLinkedList",
+              "type": "LibraryLinkingChange"
+            }
+          ]
+        },
+        "versionDelta": {
+          "storage": "=",
+          "major": "=",
+          "minor": "+1",
+          "patch": "+1"
+        }
+      },
+      "Governance": {
+        "changes": {
+          "storage": [],
+          "major": [],
+          "minor": [
+            {
+              "contract": "Governance",
+              "signature": "revokeVotes()",
+              "type": "MethodAdded"
+            }
+          ],
+          "patch": [
+            {
+              "contract": "Governance",
+              "type": "DeployedBytecode"
+            },
+            {
+              "contract": "Governance",
+              "dependency": "Proposals",
+              "type": "LibraryLinkingChange"
+            },
+            {
+              "contract": "Governance",
+              "dependency": "IntegerSortedLinkedList",
+              "type": "LibraryLinkingChange"
+            }
+          ]
+        },
+        "versionDelta": {
+          "storage": "=",
+          "major": "=",
+          "minor": "+1",
+          "patch": "-1"
+        }
+      },
+      "LockedGold": {
+        "changes": {
+          "storage": [],
+          "major": [],
+          "minor": [
+            {
+              "contract": "LockedGold",
+              "signature": "getPendingWithdrawal(address,uint256)",
+              "type": "MethodAdded"
+            }
+          ],
+          "patch": [
+            {
+              "contract": "LockedGold",
+              "type": "DeployedBytecode"
+            },
+            {
+              "contract": "LockedGold",
+              "dependency": "Signatures",
+              "type": "LibraryLinkingChange"
+            }
+          ]
+        },
+        "versionDelta": {
+          "storage": "=",
+          "major": "=",
+          "minor": "+1",
+          "patch": "0"
+        }
+      },
+      "MetaTransactionWallet": {
```

Comment:
Should this have been deployed? Not a core contract.

Reply:
It is in the previous reports unfortunately.